scripts/semantic-commits/get-current-release-data.js

```diff
@@ -17,6 +17,7 @@ const getCurrentReleaseData = (verbose = true) => {
   }
 
   verbose && console.log({ latestReleaseInfo })
+  latestReleaseInfo.versions = npmInfo.versions
```

Comment:
What's this for?

Reply:
This was an update Emily was trying out to get the verify changelog build step passing; it's since been merged into develop. I'll get this cleaned up, just some branching messiness.

Reply:
Done in upcoming commit.